src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Certificate2.cs

```diff
@@ -327,8 +327,7 @@ public string Thumbprint
         {
             get
             {
-                byte[] thumbPrint = GetCertHash();
-                return thumbPrint.ToHexStringUpper();
+                return GetCertHashString();
```

Comment:
It's nice to be able to avoid the byte[] allocation, but is there any concern that this could be a behavior change if a derived type was overriding GetCertHashString?  Otherwise, LGTM.

Reply:
It actually aligns the behavior with NetFX, which calls GetCertHashString (https://referencesource.microsoft.com/#System/security/system/security/cryptography/x509/x509certificate2.cs,930).

Reply:
Ah, that's good to hear.